Procfile~

```diff
@@ -0,0 +1 @@
+web: python journal.py
```

Comment:
Don't keep around the old save files that your editor leaves behind.  You might wish to add `*~` to your .gitignore file after deleting this.